.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,75 @@
+name: Dev Deployment Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches: ['develop']
+    paths-ignore:
+      - '**.md'
+      - '**/*.gitignore'
+      - '**/*.gitattributes'
+      - '**/*.yml'
+  pull_request:
```

Comment:
Change this so on pull request it doesn't upload to the store but just generate an APK